### DISCUSSION TITLE
Add new Feature: ISMS inventory

### DISCUSSION THREAD

**Discussion by Canela352** [dbarzin/mercator#902] [General]
Hello
The Mercator tool is great and we are already recommending it to our clients. In these recommendations I have received specific questions and I leave them here, thank you:
Is there any possibility to add a new tab in the "GDPR View" or any other place with the following characteristics:
It is called "ISMS Information" (for example) and link could be made to all the documentation related to them: policies, processes, audit documents, etc.
With space to add data of responsible persons and linked departments.
with a link for applications, servers, buildings, processes, tasks, etc. 
In  In addition, to be able to upload documents there or have a space where the link to the location of these documents could be added.

In the "physical infrastructure view" space, in addition to cameras and access badges, Is it possible to add other very specific things that may be important for a type of business, such as "air conditioners or fan" i or add a specific type of physical?

Thanks

**Comment by dbarzin** [comment]
1. Adding an "ISMS Information" Tab:

We can certainly explore the possibility of adding an "ISMS Information" tab or similar functionality. However, it’s worth noting that we already have a "Security Controls" section that may cover many of the same needs. You could add the relevant ISMS documentation (policies, processes, audit documents) to this section.

Regarding your request to link applications, servers, buildings, processes, tasks, etc., it’s important to consider whether this level of granularity is truly useful. Often, security procedures apply broadly to all systems or none, so listing each object might not always be necessary. However, if you see specific value in this, we can discuss adding these links as optional metadata for those cases where it's essential.

2. Physical Infrastructure View:

In the "Physical Infrastructure View," the feature you're asking about—tracking additional elements like air conditioners or fans—can be easily added. In fact, the existing "cameras" and "access badges" fields were generalized into attributes that allow you to define and add your own specific physical assets. You can customize these attributes for any infrastructure elements that are important to your business, including air conditioners, generators, or other equipment.

Let me know if you need further clarification or have additional questions!

**Reply by Canela352** [reply]
Thank you for anwer. 
One more question, is it possible to re-mane some ongles in the "system de information" metier?

Best regards

Carmen

**Reply by dbarzin** [reply]
Thank you for your question. Unfortunately, it is not possible to rename the tabs directly, as these elements are not configurable within the system. However, since the project is Open Source, you are free to modify the code to better suit your specific needs.

**Comment by Canela352** [comment]
Mr Barzin

As a sugestion: a direct link between task and activities without having to go through operation. 
Please receive my excuse for bother you. 
Best regards

**Comment by dbarzin** [comment]
Could you be more precise in your request, where do you want this link between task and activities to be placed ?

**Reply by Canela352** [reply]
Hello
I hope you had had a nice weekend.
thank you for your question; Certains onglets ne peuvent être joints qu'à des onglets très spécifiques, par exemple : les tâches ne peuvent être jointes qu'à des opérations et les informations ne peuvent être jointes qu'à des activités.
J'aimerais avoir la flexibilité de pouvoir relier les tâches et les informations directement à d'autres points, par exemple les tâches aux activités et les informations aux opérations ou aux activités.

Best regards
